src/utils/data/applicationFilters.ts

```diff
@@ -0,0 +1,128 @@
+import { DocumentNode } from 'graphql'
+import {
+  ApplicationOutcome,
+  ApplicationStatus,
+  AssignerAction,
+  ReviewerAction,
+} from '../generated/graphql'
+import getApplicantFilterList from '../graphql/queries/applicationListFilters/getApplicantFilterList'
+import getAssignerFilterList from '../graphql/queries/applicationListFilters/getAssignerFilterList'
+import getOrganisationFilterList from '../graphql/queries/applicationListFilters/getOrganisationFilterList'
+import getReviewersFilterList from '../graphql/queries/applicationListFilters/getReviewersFilterList'
+import getStageFilterList from '../graphql/queries/applicationListFilters/getStageFilterList'
+import { FilterDefinitions, GetFilterListQuery } from '../types'
+
+const constructFilterListQuery = (query: DocumentNode, queryMethod: string) => {
+  const getListQuery: GetFilterListQuery = ({ searchValue, filterListParameters }) => {
+    const { templateCode = '' } = filterListParameters || {}
+    return {
+      query,
+      variables: {
+        searchValue,
+        templateCode,
+      },
+      resultExtractor: (resultQuery) => ({
+        list: resultQuery?.[queryMethod]?.nodes || [],
+        totalCount: resultQuery?.[queryMethod]?.totalCount || 0,
+      }),
+    }
+  }
+  return getListQuery
+}
+
+export const APPLICATION_FILTERS: FilterDefinitions = {
+  lastActiveDate: {
+    type: 'date',
+    title: 'check',
```

Comment:
Found the `check` here @CarlosNZ  : )

Reply:
Thanks, this is fixed in upstream branch, but I've fixed it here too